[kroki-fetch-diagram] cannot find module 'mkdirp'

I've installed asciidoctor-kroki in node.js (windows) together with reveal.js. I set the attribute `:kroki-fetch-diagram: true` to fetch the diagram during convert to have the presentation for offline usage later. I've registered it with `own convert-slides.js`:

```js
// Load Asciidoctor.js and the reveal.js converter
var asciidoctor = require('@asciidoctor/core')()
var asciidoctorRevealjs = require('@asciidoctor/reveal.js')
asciidoctorRevealjs.register()

// Load asciidoctor-kroki
var kroki = require('asciidoctor-kroki')
kroki.register(asciidoctor.Extensions)

// Convert the document 'presentation.adoc' using the reveal.js converter
var options = { safe: 'unsafe', backend: 'revealjs' }
asciidoctor.convertFile('presentation.adoc', options)
```
Now it coplains aboud missing dependency

```console
Skipping plantuml block. Cannot find module 'mkdirp'
Require stack:
- d:\Project\docu\node_modules\asciidoctor-kroki\dist\node\node-fs.js
- d:\Project\docu\node_modules\asciidoctor-kroki\dist\node\fetch.js
- d:\Project\docu\node_modules\asciidoctor-kroki\dist\node\asciidoctor-kroki.js
- d:\Project\docu\convert-slides.js
```

--- a/package.json
+++ b/package.json
@@ -17,6 +17,7 @@
     "dist:node": "mkdir -p dist/node && cp src/**.js dist/node/"
   },
   "dependencies": {
+    "mkdirp": "^1.0.4",
     "pako": "^1.0.10",
     "rusha": "^0.8.13",
     "unxhr": "1.0.1"

